static/locales/en.client.json

```diff
@@ -2236,5 +2237,31 @@
         "Skip leading whitespace": "Skip leading whitespace",
         "Start with row": "Start with row",
         "Character encoding. See [the supported codecs]({{link}})": "Character encoding. See [the supported codecs]({{link}})"
+    },
+    "OpenAccessibilityModal": {
+        " or ": " or ",
+        "\"Regions\" are what we call the different parts of the user interface:": "\"Regions\" are what we call the different parts of the user interface:",
+        "Accessibility": "Accessibility",
+        "Close": "Close",
+        "Finally, the right panel – or the creator panel – is only available through its own shortcut and is not included in the next and previous region cycle.": "Finally, the right panel – or the creator panel – is only available through its own shortcut and is not included in the next and previous region cycle.",
+        "Focus on other parts of the user interface using the following shortcuts:": "Focus on other parts of the user interface using the following shortcuts:",
+        "High contrast theme": "High contrast theme",
+        "Keyboard navigation": "Keyboard navigation",
+        "On a document page, keyboard navigation is first locked on the current widget.": "On a document page, keyboard navigation is first locked on the current widget.",
+        "On document pages, each [widget]({{supportPageUrl}}) is a region that can receive focus.": "On document pages, each [widget]({{helpWidgets}}) is a region that can receive focus.",
```

Comment:
The placeholders got out of sync here, and the `widget` link appears broken because of it.